subprojects/harfbuzz.wrap

```diff
@@ -1,7 +1,7 @@
 [wrap-git]
 directory = harfbuzz
 url = https://github.com/harfbuzz/harfbuzz
-revision = master
```

Comment:
This should be fixed in the libass port, so now that Meson has autopromotion I think we can get rid of the libass-specific wrapfiles entirely?